src/test/expanded-api.test.js

```diff
@@ -62,6 +62,19 @@ describe('expanded api', () => {
       expect(Comp2.styledComponentId).toBe('Comp2-OMGLOL')
       expect(shallow(<Comp2 />).prop('className')).toMatch(/Comp2-OMGLOL/)
     })
+
+    it('should work with `.extend` and `.withComponent`', () => {
```

Comment:
Can we separate the `.extend` from the `.withComponent` tests? That should make these tests a bit shorter :)